### DISCUSSION TITLE
Welcome to dataimporthandler Discussions!

### DISCUSSION THREAD

**Discussion by chatman** [rohitbemax/dataimporthandler#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emajar** [comment]
Hello. I am trying to implement DIH in solr 9.4 . Unfortunately i dont find any instructions how to implement this in Windows environment. Any ideas... where do I put jar files? data-config.xml? does it show up in Admin UI for solr 9.0+ as before?

**Reply by aadrian** [reply]
> Unfortunately i dont find any instructions how to implement this in Windows 

@emajar I don't think that DIH  9.x was ever tested with Windows so far. 
Could you use Vagrant or some other virtualization to be able to try it with Linux? 
(since if that would work, than it would be more about changing some paths/ bash scripts)

**Reply by emajar** [reply]
@aadrian Thanks for quick reply, If I knew the implementation structure, probably i could change paths etc. I have been searching for DIH alternative, but seems there is none for solr 9.x  How are people indexing after the DIH deprecation? Any ideas would help

**Reply by aadrian** [reply]
@emajar 
Here's the old WIKI https://cwiki.apache.org/confluence/display/solr/dataimporthandler
(most stuff is still valid)

> How are people indexing after the DIH deprecation? 

DIH is not deprecated, but just decoupled from the original project (so it's not released together).  Everything else works like before, with only a certain delay, since it usually takes a few days to update and release DIH each time Solr gets a release.
